career/roles.md

```diff
@@ -58,7 +58,7 @@ The following descriptions are indications of what is expecting of a person havi
 - Budget
 - Strategy planning
 
-## Jesper Sørensen - Principal Engineer
```

Comment:
I can see how it seems out of place to mention his name here.
However, we currently only have one principal engineer. So it is not wrong to have his name here.